src/snowflake/cli/plugins/nativeapp/exceptions.py

```diff
@@ -54,18 +54,18 @@ def __init__(self, item: str, expected_owner: str, actual_owner: str):
         )
 
 
-class MissingPackageScriptError(ClickException):
-    """A referenced package script was not found."""
+class MissingScriptError(ClickException):
+    """A referenced script was not found."""
 
     def __init__(self, relpath: str):
-        super().__init__(f'Package script "{relpath}" does not exist')
+        super().__init__(f'Script "{relpath}" does not exist')
 
 
-class InvalidPackageScriptError(ClickException):
-    """A referenced package script had syntax error(s)."""
+class InvalidScriptError(ClickException):
+    """A referenced script had syntax error(s)."""
 
     def __init__(self, relpath: str, err: jinja2.TemplateError):
-        super().__init__(f'Package script "{relpath}" is not a valid jinja2 template')
+        super().__init__(f'Script "{relpath}" does not contain a valid template')
         self.err = err
```

Comment:
@sfc-gh-melnacouzi  Despite making the `err` available inside the error, will the user actually see what went wrong? Or will they just get "script does not contain a valid template" and no additional information about the error?

Reply:
Good point. I have updated the message to be more informative.

Reply:
I believe it's the latter, but we should indeed confirm. Why not fix this here then? It would be informative for a user if we gave them the Jinja error, and while we're fixing bugs, might as well fix all known bugs in the current logic...